Cargo.toml

```diff
@@ -49,6 +49,10 @@ version = "0.5.0-dev"
 path = "protocol"
 version = "0.5.0-dev"
 
+[dependencies.librespot-oauth]
```

Comment:
Good idea to extract it to a new crate. Should require an update to https://github.com/librespot-org/librespot/blob/dev/publish.sh too.